mri/reconstructors/utils/extract_sensitivity_maps.py

```diff
@@ -45,9 +46,14 @@ def extract_k_space_center_and_locations(data_values, samples_locations,
     is_fft: bool default False
         Checks if the incoming data is from FFT, in which case, masking
         can be done more directly
+    density_comp: np.ndarray default None
+        The density compensation for kspace data in case it exists and we
+        use density compensated adjoint for Smap estimation
     Returns
     -------
-    The extracted center of the k-space
+    The extracted center of the k-space. If the density compensators are
```

Comment:
Well since you are here you might as well correct the docs in full since we do not only return the center of the k-space but also the locations.

Also don't forget to indicate the dtypes.

Reply:
Sorry the dtypes? I have changed the sentence to: 
```
The extracted center of the k-space, i.e. both the kspace locations and kspace values.
```

Arent the return dtypes same as input?

Reply:
Well surely, but it's still good to remind that